llvm-project/Makefile

```diff
@@ -35,3 +34,6 @@ clang: epoch3
 	$(DOCKER) rm llvm-project || true
 	$(DOCKER) create --name llvm-project $(IMAGE):stage3-$(HOST_ARCH)
 	$(DOCKER) cp llvm-project:/usr/local/bin/clang-14 clang
+
+test: clang
```

Comment:
I think `epoch3` would be a better prerequisite, as we are testing the container, not the binary?